nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -178,6 +179,8 @@
   // Unmatched, used for when we only want full checker suppressions to work
   static final String CORE_CHECK_NAME = "NullAway.<core>";
 
+  private static final Supplier<Type> JAVA_LANG_VOID = Suppliers.typeFromString("java.lang.Void");
```

Comment:
There's also `Suppliers.VOID_TYPE`, which looking at the implementation may be more efficient. (Uses `state.getSymtab().voidType`.)

Reply:
Oh, or is that the non-boxed variant? In that case, nevermind.

Reply:
There is `Suppliers#JAVA_LANG_VOID_TYPE` which I think does what we want.  Thanks for the tip!

Reply:
Missed that one :facepalm:. Nice!

Reply:
In the end I realized we really meant to be checking for the unboxed type!  🙂 And there is an `isPrimitiveOrVoid()` method for doing that.